docs/scarpet/Full.md

```diff
@@ -1109,7 +1109,7 @@ for(range(1000000,1100000),check_prime(_))  => 7216
 
 From which we can learn that there is 7216 primes between 1M and 1.1M
 
-### `while(cond, limit)`
+### `while(cond, limit,expr)`
```

Comment:
Full.md is auto-generated from the individual documentation files.

Reply:
Should I make another commit, or just leave this be?

Reply:
editing Full has no effect - doesn't matter too much